### PR TITLE
Feature/celery send email notification if task fails

### PR DESCRIPTION
**Pull request by juuso-j**
# Send email notification if celery task fails.

-----------------------------------------------------------------------------------------------
#### Files changed
 1. eco_counter/tasks.py
 2. iot/tasks.py
 3. mobility_data/tasks.py
 4. ptv/tasks.py
 5. smbackend_turku/tasks.py
     * Replace @shared_task with @shared_task_email.
 6. smbackend/settings.py
     * Add email settings and celery admin group. 
7. smbackend/utils.py
    * Add function(decorator) that sends email if and exception is raised in wrapped function.

### CODE REVIEW

File: smbackend/settings.py [hunk 1]
```diff
@@ -217,6 +217,11 @@ def gettext(s):
 # Shortcut generation URL template
 SHORTCUTTER_UNIT_URL = env("SHORTCUTTER_UNIT_URL")
 
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = "smtp.turku.fi"
+EMAIL_HOST_USER = "varaamo@turku.fi"
```
Comment (ezkat) [inline smbackend/settings.py:222]: Switch to env value and request different address for this project

File: smbackend/utils.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        except Exception:
+            notify_emails = get_emails_to_notify()
+            if len(notify_emails) > 0:
+                subject = "Celery task failure"
+                message = f"Function: {str(func)} \n"
+                message += traceback.format_exc()
+                send_mail(
+                    subject,
+                    message,
+                    "varaamo@turku.fi",
```
Comment (ezkat) [inline smbackend/utils.py:39]: Switch to env value and request different address for this project

**Review by ezkat — CHANGES_REQUESTED** [review]

File: smbackend/settings.py (hunk 1 above)
Comment (juuso-j) [reply smbackend/settings.py]: Fixed

File: smbackend/utils.py (hunk 2 above)
Comment (juuso-j) [reply smbackend/utils.py]: Fixed

**Review by ezkat — APPROVED** [review]

**Pull request merged**